tests/torch/test_nncf_network.py

```diff
@@ -147,24 +148,36 @@ def test_weight_normed_modules_are_replaced_correctly():
     assert len(wrapped_conv._forward_pre_hooks) == 1
 
 
-@register_module()
```

Comment:
Why did you do the following changes?
```
@register_module()
class RegisteredModuleOfUser(ModuleOfUser):
    pass
```

Reply:
This is needed to check that non registered user module which calls function (like `F.conv2d`) isn't wrapped by NNCF. I forgot to add the check, will do shortly

Reply:
I'd add clarification to the name of different modules: like `RegisteredModuleWithOperation`, `RegisteredModuleWithNestedModule`, `UnregisteredModuleWithOperation`